packages/patterns-react/src/patterns/ListSection/ListSectionGroup.js

```diff
@@ -24,14 +25,22 @@ const { prefix } = settings;
  */
 const ListSectionGroup = ({ listGroup: { title, lists } }) => {
```

Comment:
Is `ListSectionGroup`, and `ContentGroup` kind of the same thing? Does it make sense to remove this file, and use `ContentGroup similarly to how `ListSectionGroup` was being used in `ListSection`?

Reply:
Right...it seems like `ListSectionGroup` can be removed entirely and just use `ContentGroup` directly?

Reply:
@jeffchew yeah, that's what I was trying to suggest. Sorry if it came out a little confusing.